Test fails when running with non-default testdata-folder

The test `test_ensemble_set_model.py::test_single_ensemble` fails when running with a non-default `testdata-folder`

**How to reproduce**
Clone `webviz-subsurface-testdata` to a folder located side-by-side with the `webviz-subsurface` folder.
Run test with `webviz-subsurface` as current folder, eg:
`pytest --headless --forked -k "test_ensemble_set_model.py" --testdata-folder ../webviz-subsurface-testdata ./tests`

The following test fails:
`tests/unit_tests/model_tests/test_ensemble_set_model.py:9 test_single_ensemble`

**Error:**
```
testdata_folder = PosixPath('../webviz-subsurface-testdata')

    @pytest.mark.usefixtures("app")
    def test_single_ensemble(testdata_folder):

        emodel = EnsembleSetModel(
            ensemble_paths={
                "iter-0": str(
                    Path(
                        testdata_folder / "reek_history_match" / "realization-*" / "iter-0"
                    )
                )
            }
        )
>       assert emodel.ens_folders == {
            "iter-0": Path("webviz-subsurface-testdata") / "reek_history_match"
        }
E       AssertionError: assert {'iter-0': Po...story_match')} == {'iter-0': Po...story_match')}
E         Differing items:
E         {'iter-0': PosixPath('../webviz-subsurface-testdata/reek_history_match')} != {'iter-0': PosixPath('webviz-subsurface-testdata/reek_history_match')}
E         Use -v to get the full diff

tests/unit_tests/model_tests/test_ensemble_set_model.py:21: AssertionError
```

--- a/tests/unit_tests/model_tests/test_ensemble_set_model.py
+++ b/tests/unit_tests/model_tests/test_ensemble_set_model.py
@@ -19,7 +19,7 @@ def test_single_ensemble(testdata_folder):
         }
     )
     assert emodel.ens_folders == {
-        "iter-0": Path("webviz-subsurface-testdata") / "reek_history_match"
+        "iter-0": Path(testdata_folder) / "reek_history_match"
     }
     assert len(emodel.ensembles) == 1
     smry = emodel.load_smry()

